core/kits/assets/js/hooks/ui/document/elements/settings/update-breakpoints-preview.js

```diff
@@ -19,7 +19,35 @@ export class KitUpdateBreakpointsPreview extends $e.modules.hookUI.After {
 		const { settings } = args;
 
 		if ( settings.active_breakpoints ) {
+			// Updating the config is necessary, even if the page has to be reloaded for these settings to take place,
```

Comment:
Check if the following code is necessary

Reply:
Only some of it is necessary. I sent in a fix removing the unnecessary code.